src/SignalR/common/Protocols.MessagePack/src/Protocol/MessagePackHubProtocol.cs

```diff
@@ -71,13 +72,13 @@ private void SetupResolver(MessagePackHubProtocolOptions options)
                 // check if the user customized the resolvers
                 if (options.FormatterResolvers[i] != SignalRResolver.Resolvers[i])
                 {
-                    _resolver = new CombinedResolvers(options.FormatterResolvers);
+                    var resolver = new CombinedResolvers(options.FormatterResolvers);
```

Comment:
Looks like you missed creating the serializer options

Reply:
classic pabkac

Reply:
Use the Microsoft.CodeAnalysis.BannedApiAnalyzers package and add a banned api file to ban all APIs that do not include the options in order to stay on the safe side. 
https://github.com/neuecc/MessagePack-CSharp/issues/534 tracks making this easier.